docs/serving/samples/knative-routing-go/index.md

```diff
@@ -260,6 +260,41 @@ The Gateway proxy checks the updated host, and forwards it to `Search` or
 
 ![Object model](./images/knative-routing-sample-flow.png)
 
+## Using internal services and `"httpProtocol": "Redirected"`
+
+Using the above approach, services will be available using two entrypoints into the cluster:
+The original ones provided by Knative Serving (`search-service.default.example.com` and `login-service.default.example.com`),
+as well as the additional entrypoints `example.com/search` and `example.com/login`
+provided by the manually added VirtualService (`entry-route`).
+
+To make sure your service can only be reached via the manually created
+VirtualService, you can add the label `networking.knative.dev/visibility: cluster-local`
+to the KNative Service definitions, and route traffic over
```

Comment:
```suggestion
to the Knative Service definitions, and route traffic over
```
One more `KNative`.

Reply:
Sorry, I totally overread this one. Thanks for pointing me to it :)